src/runner.ts

```diff
@@ -39,21 +40,29 @@ const reporterConfPath = `${__dirname}/reporter.conf.js`;
 async function runCypress(testFolder: string, browser: string) {
 
 	try {
-		await cleanUpResults(`/${testFolder}`);
+		await cleanUpResults(testFolder);
 		await exec(['cypress run',
 			`--browser ${browser}`,
-			`--spec ${CY_TESTS_BASE_PATH}/${testFolder}/${CY_SPEC_FILES_PATTERN}`,
+			`--spec ${joinTestFolder(CY_INTEGRATION_FOLDER, testFolder)}/${CY_SPEC_FILES_PATTERN}`,
+			`--reporter cypress-multi-reporters`,
 			`--reporter-options configFile=${reporterConfPath}`,
 		].join(' '), {
+			preferLocal: true,
 			env: {
 				CY_TEST_FOLDER: path.resolve(CY_RESULTS_PATH, testFolder),
 			},
 		});
 	}
+	catch (e) {
+		console.error(e);
```

Comment:
Nitpick: wouldn't it be enough to just log / just rethrow?

Reply:
- log is needed because executor hides error message, so you won't no what went wrong
- re-throw is needed because that indicates failed state